webpack/components/extensions/HostDetails/HostDetailsActions.js

```diff
@@ -1,2 +1,13 @@
+import HOST_DETAILS_KEY from './HostDetailsConstants';
+
 const hostIdNotReady = { type: 'NOOP_HOST_ID_NOT_READY' };
+
+export const getHostDetails = ({ hostname }) => ({
+  type: 'API_GET',
+  payload: {
+    key: HOST_DETAILS_KEY,
+    url: `/api/hosts/${hostname}`,
```

Comment:
This action was left over from an intermediate commit, but I decided that it still makes more sense for the host collections modal to import the action from here rather than just doing it in that component. This way the action can be easily reused in the future.